sycl/include/CL/sycl/item.hpp

```diff
@@ -20,6 +20,8 @@ struct Builder;
 }
 template <int dimensions> struct id;
```

Comment:
Probably everything is a `class` instead of a `struct` now

Reply:
id is still a struct